docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md

```diff
@@ -76,7 +76,11 @@ Defines one or more switches used by the <xref:System.AppContext> class to provi
 |`Switch.System.IO.Compression.ZipFile.`<br/>`UseBackslash`|Uses the backslash ("\\") rather than the forward slash ("/") as the path separator in the <xref:System.IO.Compression.ZipArchiveEntry.FullName%2A?displayProperty=nameWithType> property. For more information, see  [Mitigation: ZipArchiveEntry.FullName Path Separator](~/docs/framework/migration-guide/mitigation-ziparchiveentry-fullname-path-separator.md).|.NET Framework 4.6.1|  
 |`Switch.System.IO.`<br/>`UseLegacyPathHandling`|Controls whether legacy path normalization is used and URI paths are supported by the <xref:System.IO.Path.GetDirectoryName%2A?displayProperty=nameWithType> and <xref:System.IO.Path.GetPathRoot%2A?displayProperty=nameWithType> methods. For more information, see [Mitigation: Path Normalization](~/docs/framework/migration-guide/mitigation-path-normalization.md) and [Mitigation: Path Colon Checks](~/docs/framework/migration-guide/mitigation-path-colon-checks.md).|.NET Framework 4.6.2|  
 |`Switch.System.`<br/>`MemberDescriptorEqualsReturnsFalseIfEquivalent`|Controls whether a test for equality compares the <xref:System.ComponentModel.MemberDescriptor.Category%2A?displayProperty=nameWithType> property of one object with the <xref:System.ComponentModel.MemberDescriptor.Description%2A?displayProperty=nameWithType> property of the second object. For more information, see [Mitigation: MemberDescriptor.Equals](~/docs/framework/migration-guide/mitigation-memberdescriptor-equals.md).|.NET Framework 4.6.2|  
-|`Switch.System.Net.`<br/>`DontEnableSchUseStrongCrypto`|Controls whether the <xref:System.Net.ServicePointManager?displayProperty=nameWithType> and <xref:System.Net.Security.SslStream?displayProperty=nameWithType> classes can use the SSL 3.0 protocol. For more information, see [Mitigation: TLS Protocols](~/docs/framework/migration-guide/mitigation-tls-protocols.md).|.NET Framework 4.6|  
+|`Switch.System.Net.`<br/>`DontCheckCertificateEKUs`|Disables certificate EKU OID validation.|.NET Framework 4.6|
```

Comment:
@davidsh we usually have to define what acronyms are on the first use. Should we change this from EKU OID to extended key usage (EKU) object identifier (OID) or does it get too noisy?

Reply:
>Should we change this from EKU OID to extended key usage (EKU) object identifier (OID)

I was leaving that up to you.  We probably should be standardizing how we talk about these kinds of terminology.  EKU OID are .NET concepts:
https://msdn.microsoft.com/en-us/library/system.security.cryptography.x509certificates.x509enhancedkeyusageextension(v=vs.110).aspx

"An enhanced key usage (EKU) extension is a collection of object identifiers (OIDs) that indicate the ..."

So, feel free to expand how we are describing this.

Reply:
Got the wrong name, sorry. Ok, the standard process is to expand them on first use, so I'll make that fix here. Thanks @davidsh!